content/swiftui/concepts/viewmodifier/terms/underline/underline.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: '.underline()'
+Description: 'A ViewModifier that applies an underline style to text characters in a View.'
+Subjects:
+   - 'Mobile Development'
+   - 'iOS'
+Tags:
+   - 'SwiftUI'
+   - 'SwiftUI Modifiers'
+CatalogContent:
+   - 'learn-swift'
+   - 'paths/build-ios-apps-with-swiftui'
+---
+
+**.underline()** is a [`ViewModifier`](https://www.codecademy.com/resources/docs/swiftui/viewmodifier) that applies an underline style to text characters in a [`View`](https://www.codecademy.com/resources/docs/swiftui/views).
```

Comment:
```suggestion
**.underline()** is a `ViewModifier` that applies an underline style to text characters in a [`View`](https://www.codecademy.com/resources/docs/swiftui/views).

```

Generally we don't link out to parent entries because the normal Docs navigation will provide those links for us at the top of the entry.